packages/mds-agency/utils.ts

```diff
@@ -360,7 +359,7 @@ export async function readPayload(device_id: UUID): Promise<VehiclePayload> {
   try {
     payload.device = await db.readDevice(device_id)
   } catch (err) {
-    logger.error(err)
+    AgencyLogger.error(err)
```

Comment:
Seems like just logging out the error without accompany text is less than ideal.